ExamenProeveUnity/Assets/Scripts/Runtime/Player/PlayerGrab.cs

```diff
@@ -43,8 +49,6 @@ private void HandleMouseRelease()
 
         private void HandleMousePressed()
```

Comment:
new input system uses more than mouse presses
change name to something like 'HandleHandPressed()'